plugins/dev-scripts/config/webpack.config.js

```diff
@@ -55,6 +57,16 @@ module.exports = (env) => {
     target = 'node';
   }
 
+  // Add 'dist' to the end of the blockly module alias if we have acquired
+  // blockly from git instead of npm.
+  let blocklyAliasSuffix = '';
+  const blocklyDependency =
+    (packageJson.dependencies && packageJson.dependencies['blockly']) ||
+    (packageJson.devDependencies && packageJson.devDependencies['blockly']);
+  if (blocklyDependency && blocklyDependency.indexOf('git://') === 0) {
```

Comment:
This is not guaranteed to be a good check: https://docs.npmjs.com/files/package.json#github-urls

However, we could just require that you use this prefix. nbd.

Reply:
Yeah, I didn't want to make the check not semver, because you can technically do tags and things like `latest` in that field, so it's simpler to just limit it to `git://` for this use case.